src/app/routes.py

```diff
@@ -21,27 +28,43 @@ def dashboard():
     return render_template('dashboard.html')
 
 
-@main.route('/query', methods=['GET', 'POST'])
+@main.route('/query')
 @oidc.require_login
 def query():
-    auths = ['FOO', 'BAR', 'PUBLIC', 'PRIVATE']
+    cmd = 'authorization -c [PLACEHOLDER] -k [PLACEHOLDER]'.split(' ')
+    whoami = CliRunner().invoke(dwv_entry_point, cmd, standalone_mode=False).return_value
+    auths = whoami['proxied_users'][0]['auths']
+
+    return render_template('query.html', auths=auths)
+
 
+@main.route('/query_results', methods=['GET', 'POST'])
+@oidc.require_login
+def query_results():
     if request.method == 'POST':
-        query_name = request.form.get('query_name')
+        query_name = request.form.get('query_name') or uuid4()
```

Comment:
```suggestion
        query_name = request.form.get('query_name', uuid4())
```

Reply:
Just checked. This does not work because it does retreive something. It's just blank.